app/views/received_kudos_mailer/received_kudos_notification.html.erb

```diff
@@ -0,0 +1,11 @@
+<strong>Congratulations, <%= @receiver.name %>!</strong>
+<p>
+  You just received some Kudos from <%= @giver.name %>!
+</p>
+<p>
+  Click on this link to read it: <%= @url %>
```

Comment:
`@url` is a string, so this will probably just output `http://kudos.factual.com`. Gmail does some magic and automatically links things that look like URLs, but I think this should more explicitly create the anchor tag. i.e...

````erb
<a href='<%= @url %>'><%= @url %></a>
````

Or maybe not even displaying the raw URL...

````erb
<a href='<%= @url %>'>Click here to view your kudo</a>
````